test/scenarios/bookshop/search.test.js

```diff
@@ -0,0 +1,19 @@
+const cds = require('../../cds.js')
+const bookshop = require('path').resolve(__dirname, '../../bookshop')
```

Comment:
Needs to be included in the bookshop/index.js file. So didn't actually run in the GitHub action.